devtools/src/arcs-bug-report.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+import {PolymerElement} from '../deps/@polymer/polymer/polymer-element.js';
+import {html} from '../deps/@polymer/polymer/lib/utils/html-tag.js';
+import {MessengerMixin} from './arcs-shared.js';
+import '../deps/jszip/dist/jszip.js';
+
+/**
+ * Saves/loads a bug report which contains debug messages sent to devtools.
+ */
+class ArcsBugReport extends MessengerMixin(PolymerElement) {
+  static get template() {
+    return html`
+    <style include="shared-styles">
+      :host {
+        display: flex;
+        flex-direction: column;
+      }
+    </style>
+    <div class="wrapper">
+      <input type="file" id="fileElem" style="display:none" on-change="_loadBugReport">
+      <button id="fileSelect">Load Bug Report</button>
+      <button type="button" on-click="_saveBugReport">Save Bug Report</button>
```

Comment:
Maybe `<iron-icon title="Download Bug Report" icon="bug-report" on-click="fileBugReport"></iron-icon>` ?

It would draw a "bug" icon:
![Screen Shot 2019-09-24 at 10 22 18 am](https://user-images.githubusercontent.com/26159485/65534934-446dbc80-deb5-11e9-9ccd-ef0928ff27a4.png)

Reply:
I added one icon to the left of these two buttons, which should make it clear that this section is bug related...